docs/reference/8-admin-app/technical-articles/powershell-installation-ods-api-v33.md

```diff
@@ -0,0 +1,159 @@
+# PowerShell Installation (ODS / API v3.3)
+
+# Before You Install
+
+This section provides general information you should review before installing
+the Ed-Fi ODS / API Admin App.
+
+## Prerequisites
+
+The following are required to install the Admin App:
+
+* The Admin App provides an interface to administer an Ed-Fi ODS / API.
+  Understandably, you must have an Ed-Fi ODS / API deployed and operational
+  before you can use the Admin App. The ODS / API must be an On-Premises
+  Installation or an Azure Cloud ODS from the Ed-Fi Exchange.
+* You must have an Ed-Fi license to use the Admin App. If you have an
+  installation of the ODS / API, you already have a license. The Ed-Fi License
+  is free and available online. If you haven't done so already, visit
+  the [Ed-Fi.org licensing
+  section](https://www.ed-fi.org/getting-started/license-ed-fi-technology/) for
+  details and a link to request a license.
+* Admin App authentication will work via Single Sign-On using either Active
+  Directory or Active Directory for Azure depending on deployment.
+* Download and install the Microsoft IIS URL Rewrite
+  Tool: [https://www.iis.net/downloads/microsoft/url-rewrite](https://www.iis.net/downloads/microsoft/url-rewrite) if
+  it is not already available (this may require computer restart).
+
+## Required Information
+
+You will need the following information to complete this installation:
+
+* The location of your Ed-Fi ODS / API.
+* Administrator access and credentials for either on-premises or Azure
+  environment with target Ed-Fi ODS / API.
+
+# Installation Instructions
+
+This section provides step-by-step instructions for installation. The specific
+steps are different depending on the deployment model and version of your Ed-Fi
+ODS / API.
+
+## Compatibility & Supported ODS / API Versions
+
+Currently, the ODS / API Admin App can be installed for use with the Ed-Fi ODS /
+API v3.3. See the [Ed-Fi Technology Version
+Index](https://edfi.atlassian.net/wiki/spaces/ETKB/pages/20875717/Ed-Fi+Technology+Version+Index) for
```

Comment:
Please replace this with `[Ed-Fi Technology Suite Supported Versions]()` ... well, I'm not sure of the relative path just reading it here. Needs to link to `supported-versions.md` under the roadmap folder.